pkg/kine/drivers/dqlite/dqlite.go

```diff
@@ -36,7 +37,7 @@ type opts struct {
 	pollInterval    time.Duration
 }
 
-func New(ctx context.Context, datasourceName string, tlsInfo tls.Config) (server.Backend, error) {
+func New(ctx context.Context, datasourceName string, tlsInfo tls.Config, acPolicyConfig generic.AdmissionControlPolicyConfig) (server.Backend, error) {
 	logrus.Printf("New kine for dqlite")
 	opts, err := parseOpts(datasourceName)
 	if err != nil {
```

Comment:
Can we keep using query parameters in the datasourceName for this? Let's try to avoid exploding the function signatures with configs, not until we know what will work well.